src/notification.c

```diff
@@ -673,7 +673,8 @@ void notification_update_text_to_render(struct notification *n)
         /* print age */
         gint64 hours, minutes, seconds;
         // Timestamp is floored to the second for display purposes -- see queues.c
-        gint64 t_delta = time_monotonic_now() - (n->timestamp - n->timestamp % S2US(1));
+        gint64 thresholded_timestamp = n->timestamp + TURN_OF_SECOND_THRESHOLD_US;
```

Comment:
I think you should subtract here. It should seem as if the notification has been sent TURN_OF_SECOND_THRESHOLD_US earlier, so that the turn of second is a little earlier

Reply:
Sorry, I notice I forgot to say that the last revision should have fixed this.